.github/workflows/build.yaml

```
@@ -8,35 +8,34 @@ on:
     
 # ! Replace DotnetProject and dotnet-project with project name in real repository
 env:
-  PROJECT_NAME: DotnetProject
-  PROJECT_NAME_LOWER: dotnet-project
+  PROJECT_NAME: Arcane.Stream.Cdm
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
   
 jobs:
   validate_commit:
     name: Validate commit
     runs-on: ubuntu-latest
     if: ${{ github.ref != 'refs/heads/main' }}
     permissions:
-      id-token: write      # required for dependabot PRs
-      pull-requests: write # required for dependabot PRs
-      contents: read       # required for dependabot PRs
+      pull-requests: write
+      contents: read
     steps:
       - uses: actions/checkout@v4
       - name: Setup .NET
         uses: actions/setup-dotnet@v4.0.0
         with:
-          dotnet-version: 6.0.x
+          dotnet-version: 8.0.x
       - name: Restore dependencies
-        env:
-          NUGET_TOKEN: ${{ secrets.NUGET_TOKEN }}
-        run: dotnet restore
+        run:  dotnet clean && dotnet nuget locals all --clear && dotnet restore
```

```suggestion
        run:  dotnet restore
```

I think `dotnet restore` is enough here